cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2020,14 +2012,13 @@
   </property>
 
   <property>
-    <name>ssl.enabled</name>
-    <value>${ssl.external.enabled}</value>
+    <name>ssl.internal.enabled</name>
+    <value>false</value>
     <description>
-      Determines if SSL is enabled
+      Enable SSL between Router and App Fabric
```

Comment:
If we are going to use this description, let's change the description (line 2010) for `ssl.external.enabled` to match:
```
Enable SSL for external services